__tests__/endpoints.test.js

```diff
@@ -96,3 +96,41 @@ describe("/api", () => {
       });
   });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('test for 200 status on completion', () => {
+        return request(app).get(`/api/articles/3`).expect(200);
+    });
+    test('should have the correct article by id', () => {
+        return request(app).get(`/api/articles/1`).expect(200).then((result) => {
+          expect(result.body.length).toBe(1)
+        });
+    });
+    test('should have all 8 properties', () => {
+      return request(app).get(`/api/articles/1`).expect(200).then((result) => {
+        console.log(result.body)
```

Comment:
Remove console.log pelase. As this is for a single review we would actually want ab object on the response body rather than an array.